partiql-parser/src/partiql.pest

```diff
@@ -2,18 +2,102 @@ WHITESPACE = _{ " " | "\t" | "\x0B" | "\x0C" | "\r" | "\n" }
 
 COMMENT = _{ LineComment | BlockComment }
 
-BlockComment = { "/*" ~ (BlockComment | (!"*/" ~ ANY))* ~ "*/" }
-LineComment = { "--" ~ (!"\n" ~ ANY )* ~ "\n" }
+BlockComment    = _{ "/*" ~ (BlockComment | (!"*/" ~ ANY))* ~ "*/" }
+LineComment     = _{ "--" ~ (!"\n" ~ ANY )* ~ "\n" }
 
 // TODO implement a full grammar, this is a very primitive version to start
 //      working with Pest and its APIs.
 
-// Entry point for full query parsing
-Query = _{ SOI ~ Token+ ~ EOI}
+//
+// Entry point for full query parsing.
+//
+Query = _{ SOI ~ Expression ~ EOI}
+
+//
+// Expressions
+//
+
+Expression = _{
```

Comment:
Does `ValueExpression` correspond to `expr_query` from the [spec](https://partiql.org/assets/PartiQL-Specification.pdf#page=9)? And similarly for `SelectExpression` to `sfw_query`?

Reply:
Basically yes, I avoided the names because I thought them to be confusing and this is going to be the basis for the grammar in the spec eventually.